src/Core/NotificationHub/NotificationHubPushRegistrationService.cs

```diff
@@ -52,20 +52,23 @@ public async Task CreateOrUpdateRegistrationAsync(string pushToken, string devic
         switch (type)
         {
             case DeviceType.Android:
-                var featureService = _serviceProvider.GetRequiredService<IFeatureService>();
-                if (featureService.IsEnabled(FeatureFlagKeys.AnhFcmv1Migration))
+                await using (var serviceScope = _serviceProvider.CreateAsyncScope())
                 {
-                    payloadTemplate = "{\"message\":{\"data\":{\"type\":\"$(type)\",\"payload\":\"$(payload)\"}}}";
-                    messageTemplate = "{\"message\":{\"data\":{\"type\":\"$(type)\"}," +
-                                      "\"notification\":{\"title\":\"$(title)\",\"body\":\"$(message)\"}}}";
-                    installation.Platform = NotificationPlatform.FcmV1;
-                }
-                else
-                {
-                    payloadTemplate = "{\"data\":{\"data\":{\"type\":\"#(type)\",\"payload\":\"$(payload)\"}}}";
-                    messageTemplate = "{\"data\":{\"data\":{\"type\":\"#(type)\"}," +
-                                      "\"notification\":{\"title\":\"$(title)\",\"body\":\"$(message)\"}}}";
-                    installation.Platform = NotificationPlatform.Fcm;
+                    var featureService = serviceScope.ServiceProvider.GetRequiredService<IFeatureService>();
+                    if (featureService.IsEnabled(FeatureFlagKeys.AnhFcmv1Migration))
```

Comment:
I believe this migration was completed -- we should be able to remove this flag @withinfocus can you confirm?

Reply:
Yes, Kyle completed that some time ago. I just made https://github.com/bitwarden/server/pull/5030 for this.